resources/definitions/two_trees_bluer.def.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "Two Trees Bluer",
```

Comment:
This profile duplicates the TwoTrees Bluer printer already added to Cura. Could you update the existing definition instead? Changing the definition ID would require a version upgrade.